### ISSUE TITLE
Temporarily delete PN across the app

### ISSUE THREAD

**Issue by churik** [status-im/status-react#9179]
# Problem
Currently, we have a bunch of issues related to PN and enabling notifications.
Basically PNs are not working at all when app is closed and some [other issues ](https://github.com/status-im/status-react/issues?q=is%3Aissue+label%3Apush-notifications+is%3Aopen)
Also to have permission to send PNs "Enable notifications" screen was introduced. There are several issues with it currently:

1) User can't enable notifications for the app if he restores his account from seed phrase (IOS)
Literally. There is no system option. The same behaviour if user chooses "May be later" - there is no option to enable notifications later.
For Android it is enabled by default, but screen "Enable notifications" is not shown during onboarding, so the user is not aware of it.

2) System notifications are always enabling on Android (regardless of user choice)
So if the user chooses "Maybe later" - it is enabled anyway

The only flow which is working right now (it is fixed in new onboarding PR) - if user enables notifications from this screen.

In order to avoid all these issues PNs should be temporary disabled for V1 (confirmed by @rachelhamlin )

## Implementation
To delete PNs:

* [ ] 1. disable notifications in app and system popups with permissions to send notifications
* [ ] 2. remove screen "Enable notifications" from onboarding
* [ ] 3. remove "Notifications" from Profile

**Comment by rachelhamlin** [comment]
Before we do this, I think we need to put a bit of effort toward a POC for [this proposal](https://discuss.status.im/t/fixing-notifications/1379?u=rachel). 

I've heard a few technical concerns and if we are going to disable PNs entirely, let's make sure we have recorded our research into whether a more ideal implementation is feasible.

TL;DR: please no one pick up this issue next.

**Comment by rachelhamlin** [comment]
POC was conclusive and we have a WIP for improved PNs on Android, but not ready for release yet. There are myriad bugs with the current implementation, so we **do** want to disable PNs for v1 in the end. My bad - I had in mind that this was already done.

**Comment by yenda** [comment]
@rachelhamlin it is done already, in release version there should be no PN and no option to enable them

**Comment by rachelhamlin** [comment]
Okay, cool, I'm not crazy then.

**Comment by yenda** [comment]
@rachelhamlin so let's close this one?
